src/orders/components/OrderFulfillPage/OrderFulfillPage.tsx

```diff
@@ -351,19 +306,19 @@ const OrderFulfillPage: React.FC<OrderFulfillPageProps> = props => {
                             {line.productName}
                             <Typography color="textSecondary" variant="caption">
                               {line.variant?.attributes
-                                .map(attribute =>
+                                ?.map(attribute =>
                                   attribute.values
                                     .map(attributeValue => attributeValue.name)
                                     .join(", ")
                                 )
-                                .join(" / ")}
+                                ?.join(" / ")}
                             </Typography>
                           </TableCellAvatar>
                           <TableCell className={classes.colSku}>
                             {line.variant?.sku}
                           </TableCell>
                           {warehouses?.map(warehouse => {
-                            const warehouseStock = line.variant?.stocks.find(
+                            const warehouseStock = line.variant?.stocks?.find(
                               stock => stock.warehouse.id === warehouse.id
```

Comment:
getById

Reply:
Stock is taken from warehouse subfield, so getById can't be used here.